packages/main/src/Input.js

```diff
@@ -1104,7 +1104,7 @@ class Input extends UI5Element {
 	get styles() {
 		return {
 			popoverHeader: {
-				"width": `${this._inputWidth}px`,
```

Comment:
having comma at the end of object's key/value is a lint rule, to check for lint errors you can execute yarn lint in the project's root

Reply:
Done